initrd/bin/tpmr

```diff
@@ -639,15 +643,18 @@ tpm2_reset() {
 }
 tpm1_reset() {
 	TRACE "Under /bin/tpmr:tpm1_reset"
-	key_password="$1"
-
+	tpm_owner_password="$1"
+	mkdir -p "$SECRET_DIR"
+	# output tpm_owner_password to a file to be reused in this boot session until recovery shell/reboot
+	DEBUG "Caching TPM Owner Password to $SECRET_DIR/tpm_owner_password"
+	echo -n "$tpm_owner_password" > "$SECRET_DIR/tpm_owner_password"
 	# Make sure the TPM is ready to be reset
 	tpm physicalpresence -s
 	tpm physicalenable
 	tpm physicalsetdeactivated -c
 	tpm forceclear
 	tpm physicalenable
-	tpm takeown -pwdo "$key_password"
+	tpm takeown -pwdo "$tpm_owner_password"
```

Comment:
@JonathonHall-Purism as you will see in codebase, key_password has been differenciated with tpm_owner_password so its clearer what is needed when refactoring.

Reply:
@JonathonHall-Purism I have resolved the parts I was concerned about, will eave you resolve pending discussions on your side. Will also try to create discussions with GPG_AUTH. 

For context of this funded work https://nlnet.nl/project/AuthenticatedHeads/